### PR TITLE
add Naming Access_Control  Statemnts import  TODO

### PR DESCRIPTION
**Pull request by j900155**
add TODO import Statments  Access_Control Naming Main

### CODE REVIEW

File: codingstyle/README.md [hunk 1]
```diff
@@ -1,2 +1,303 @@
-# codingstyle
```
Comment (kiss5891) [inline codingstyle/README.md:1]: @ @

File: codingstyle/README.md [hunk 2]
```diff
@@ -1,2 +1,303 @@
...
+﻿﻿# codingstyle
+
+## 分號
+- 不要用分號作為一行的結束且不要用分號作為兩行之間的分隔。
+
+## 分行
+- 一行不要超過80個字，除非其中有包含長URL的註解和很長的引入敘述。
+- 不要使用反斜線來接續兩行。
+- 確保隱行被放在括弧(大、中
+- 、小)中，如有必要可以在表達式外額外新增括弧。
```
Comment (kiss5891) [inline codingstyle/README.md]: 換行？

**Comment by j900155** [comment]
fix some error

**Comment by j900155** [comment]
fix some variable name

**Comment by j900155** [comment]
fix

**Pull request merged**